src/EmbedIndex/PETimestampSizeIndexer.cs

```diff
@@ -30,8 +30,8 @@ public string ComputeIndexKey(string path, Stream fileStream)
                 StringBuilder key = new StringBuilder();
                 key.Append(filename);
                 key.Append("/");
-                key.Append(reader.Timestamp.ToString("x").ToLowerInvariant());
```

Comment:
I'm assuming you already confirmed that ToString("x") prints lower case by default?